convert_rmd_file.py

```diff
@@ -0,0 +1,76 @@
+from bs4 import BeautifulSoup
+from rpy2.robjects.packages import importr
+from tempfile import NamedTemporaryFile
+import logging
+import os
+
+from sanitize_html import sanitize
+
+
+def to_safe_html(stream):
+    binary_data = stream.read()
+    raw_rmd = binary_data.decode('ascii')
+
+    safe_rmd = _sanitize_rmd(raw_rmd)
+    raw_html = _to_html(safe_rmd)
+
+    soup = BeautifulSoup(str(raw_html), 'html.parser')
+    body_tag = soup.body
+
+    # temporarily swap body tag for a safe tagname
```

Comment:
Could you add a bit more explanation about why this swapping is needed? Also if the exact same block of code is needed for the python notebooks, would it make sense to extract this out into its own function?